script.module.dropbox/lib/dropbox/pkg_resources.py

```diff
@@ -0,0 +1,3 @@
+
+def resource_filename(*args):
+    return "trusted-certs.crt"
```

Comment:
Why hardcode the return value, instead of implementing it the correct way?

Reply:
These changes came from me. These additions came from [some advice given in the forum](https://forum.kodi.tv/showthread.php?tid=143406&pid=1221401#pid1221401). The dropbox library throws an error when pkg_resources is not present and this was the suggested work around to giving the library what it wanted without a lot of extra work. This function, returning the cert location, is all it cares about and gets the library to load and function. I've been using this workaround as part of the Backup addon for years.

Reply:
I had the following in mind, which isn't a lot of extra code, but will also work if additional `pkg_resources.resource_filename` calls are added later.
```
def resource_filename(package_or_requirement, resource_name):
    return resource_name
```

Reply:
Makes sense to me. @micahg - do you want me to make this change and do another PR on your repo so you can merge this in? I see there are other changes to consider from above as well. Just let me know if you need more from me.